pallets/did/src/mock.rs

```diff
@@ -23,6 +23,7 @@ use frame_support::{
 };
 use frame_system::EnsureSigned;
 use pallet_balances::NegativeImbalance;
+
```

Comment:
Why these new lines that I see often? Are you following some sort of convention?